### PR TITLE
changed type text to textarea

### PR DESCRIPTION
**Pull request by Ravi-kumar9347**
modified the field type text to textarea for issue 5265

### CODE REVIEW

**Comment by stevepiercy** [comment]
@Ravi-kumar9347 please read and follow [Change log entry](https://6.docs.plone.org/contributing/index.html#change-log-entry) and [Create a pull request from your fork](https://6.docs.plone.org/contributing/first-time.html#create-a-pull-request-from-your-fork) Item 2. We do not review pull requests until all GitHub checks pass.

File: packages/volto/news/5265.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Changed the field type from text to textarea in comment input field by @Ravi-kumar9347
```
[the remark this answers is not in the document]
Comment (Ravi-kumar9347) [reply packages/volto/news/5265.bugfix:1]: okay @stevepiercy

**Review by Ravi-kumar9347 — COMMENTED** [review]
updated

**Review by stevepiercy — APPROVED** [review]
Thank you! A @plone/volto-team member will need to review.

**Comment by Ravi-kumar9347** [comment]
@tiberiuichim and @sneridagh , can you check my PR once ?

**Review by davisagli — CHANGES_REQUESTED** [review]
@Ravi-kumar9347 Your fix here does not work.

I tested like this:
1. Go to http://localhost:8080/Plone/@@discussion-controlpanel and turn on "globally enable comments" and "enable editing of comments"
2. Back in Volto (http://localhost:3000/), add a page and make sure it has "allow discussion" set to "Yes"
3. Add a comment
4. Click the link to edit the comment

Expected: the input for editing the comment text is a textarea
Actual: it is still a single-line text input

Screenshot 2023-12-26 at 10 24 25 AM

In fact, all that is needed is a much simpler fix. We don't need or want a new `type`; you just need to set `widget: 'textarea'` on the `text` field in the CommentEditModal schema.

**Comment by Ravi-kumar9347** [comment]
@davisagli, i am new to the open source, so can you please help me the exact path of above image. because i only see this image. where can i find that comment sectoin.
Screenshot from 2023-12-27 01-49-01

**Comment by Ravi-kumar9347** [comment]
@stevepiercy  sorry for my things i did, can i reopen this or should i mention why i closed this pr

**Comment by stevepiercy** [comment]
@Ravi-kumar9347 do whatever you think is best. I already gave you direction in my comment in the other PR.

**Comment by Ravi-kumar9347** [comment]
@stevepiercy then would you consider the latest pr

**Pull request closed without merging**